content/03.results.md

```diff
@@ -181,7 +185,7 @@ Points are colored by `cancer_group` of the samples they represent. B, Heatmap o
 #### Prediction of _TP53_ oncogenicity and telomerase activity
 To understand the _TP53_ phenotype in each tumor, we ran a classifier previously trained on TCGA [@doi:10.1016/j.celrep.2018.03.076] to infer _TP53_ inactivation status.
 Using high-confidence SNVs, CNVs, SVs, and fusions in _TP53_ as true positive alterations, we achieved a high accuracy (AUROC = 0.85) for rRNA-depleted, stranded samples compared to randomly shuffled _TP53_ scores (**Figure {@fig:Fig5}A**). 
-The classifier did not perform well on the poly-A samples (**Figure {@fig:FigS5}A-B**), potentially due to the low number of _TP53_ altered (N = 7) and/or total poly-A samples in our dataset (N = 58) rather than library type, as a previous study demonstrated high accuracy of this classifier on another poly-A dataset [@doi:10.1016/j.celrep.2019.09.071].
+The classifier did not perform well on the poly-A samples (**Figure {@fig:FigS5}E-F**), potentially due to the low number of _TP53_ altered (N = 7) and/or total poly-A samples in our dataset (N = 58) rather than library type, as a previous study demonstrated high accuracy of this classifier on another poly-A dataset [@doi:10.1016/j.celrep.2019.09.071].
```

Comment:
N=7 is activated specifically. 22 are lost, so 29 are altered.
From:
```
library(tidyverse)
meta <- read_tsv("data/pbta-histologies.tsv", guess_max = 10000)
dat <- read_tsv("analyses/tp53_nf1_score/results/tp53_altered_status.tsv")


meta %>%
  select(Kids_First_Biospecimen_ID_RNA = Kids_First_Biospecimen_ID,
         sample_id, experimental_strategy, RNA_library) %>%
  inner_join(dat) %>%
  filter(experimental_strategy == "RNA-Seq", 
         RNA_library == "poly-A") %>%
  select(-contains("Kids_First")) %>%
  distinct() %>%
  count(tp53_altered)
```
output - 
```
# A tibble: 3 × 2
  tp53_altered     n
  <chr>        <int>
1 activated        7
2 loss            22
3 other           29
```

Reply:
good catch, thanks!